Packs/RTIR/Integrations/RTIR/README.md

```diff
@@ -120,6 +120,14 @@
 <td style="width: 589px;">Final priority of ticket</td>
 </tr>
 <tr>
+<td style="width: 132px;">member-of</td>
+<td style="width: 589px;">Ticket MembersOF links</td>
+</tr>
+<tr>
+<td style="width: 132px;">members</td>
+<td style="width: 589px;">Ticket Members links</td>
+</tr>
+<tr>
```

Comment:
please revert the file

Reply:
Reverted all files in that commit in [a445f68](https://github.com/demisto/content/pull/32381/commits/a445f68238d346069202f69ee724b03caebaca16)